src/raw/mod.rs

```diff
@@ -290,7 +280,8 @@ impl<T> Bucket<T> {
         if mem::size_of::<T>() == 0 {
             self.ptr.as_ptr() as usize - 1
         } else {
-            offset_from(self.ptr.as_ptr(), base.as_ptr())
+            //emulation of wrapping_offset_from - currently available only at nightly
+            (base.as_ptr() as usize - self.ptr.as_ptr() as usize) / mem::size_of::<T>() - 1
```

Comment:
We already have a wrapper function called `offset_from` which works on both stable and nightly.

Reply:
`self.ptr` might be constructed with out of allocation bounds, same as above conversation - should I create wrapper for `wrapping_offset_from` ?